docs/CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,179 @@
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become a Committer
+
+
+### Reviewer
+
+Description: A Reviewer is an established contributor who regularly participates in the project. Reviewers have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project.
+
+A Reviewer must meet the responsibilities and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Continues to contribute regularly, as demonstrated by having at least 30 GitHub contributions per year, as shown by Devstats. 
+* Requirements:
+    * Must have successful contributions to the project, including at least one of the following:
+        * 5 accepted PRs,
+        * Reviewed 5 PRs,
+        * Resolved and closed 10 Issues,
+        * Become responsible for a key project management area,
+        * Or some equivalent combination or contribution
+    * Must have been contributing for at least 3 months
+    * Must be actively contributing to at least one project area
+   
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
+    * Entitled to vote in the Reviewer proposal
+    * Can recommend other contributors to become a reviewer
+    * Is listed as Reviewer in the [OWNERS file](./OWNER.md) under the WasmEdge/WasmEdge repos.
+      
+
+The process for a Contributor to become a Reviewer is as follows:
+
+1. The contributor is nominated by opening a PR against the appropriate repository by the existing reviewers, committers and maintainers, which adds their GitHub username to the OWNERS file for one or more directories.
+2. At least 1/3 members of the reviewers, committers, and maintainers who are already Approvers, approve the PR.
+
+### Committer
+Description: A Committer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Committers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+
+Committers are responsible for a "specific area." This can be a specific code directory, driver, chapter of the docs, test job, event, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+
+Committers have all the rights and responsibilities of a reviewer, plus:
+
+* Responsibilities include:
+    * Reviewing most Pull Requests against their specific areas of responsibility
+    * Reviewing at least 15 PRs per year
+    * Helping other contributors become Committers
+* Requirements:
```

Comment:
Committer needs to contribute code. I think we can make it 5 PRs with code changes (not just docs or typo fixes).

Reply:
Agreed